assets/src/scss/base/_icons.scss

```diff
@@ -1,3 +1,11 @@
+// Secondary and donate buttons have a transition on the color
+// Other button styles don't need it since they don't change color on hover
+@mixin with-transition() {
```

Comment:
It seems more straightforward if we create a separate rule for the transition properties, with all selectors that need these same properties. We could use the same `transition-property` for all of them, since it includes background-color and the other 2 don't cause an issue to add to the icons.

```css
.btn,
.wp-block-button a,
.wp-block-file .wp-block-file__button,
.btn a.pdf-link,
.btn a.external-link {
    transition-property: background-color;
  transition-duration: 150ms;
  transition-timing-function: linear;
}

Reply:
External links don't need it, since only the PDF icons are applied to buttons 🤔

Reply:
Ah could be I have the wrong selector, it was just to give an example of what I meant :smile:

Reply:
I think we can just put it on `.btn` that way and don't need to do anything for the button type classes.

Reply:
Well the Button block actually doesn't have the `.btn` class but `.wp-block-button` so we'd need at least these 2

Reply:
Btw, these are variable now (one theme depends on changing these to prevent a flicker bug). You can create a second selector and also use the same `--button--` shorthand, should work fine :crossed_fingers: